readthedocs/oauth/services/gitlab.py

```diff
@@ -122,11 +122,35 @@ def sync_organizations(self):
                 remote_organizations.append(remote_organization)
 
                 for repo in org_repos:
-                    remote_repository = self.create_repository(
-                        repo,
-                        organization=remote_organization,
-                    )
-                    remote_repositories.append(remote_repository)
+                    try:
+                        # Get response for a single repository with permission data
```

Comment:
It would be good to expand this comment here about _why_ this is required and making a TODO note to optimize this --since it's data that we already fetched from `sync_repositories` method.